code/api/src/modules/user/tests/mutations.test.js

```diff
@@ -0,0 +1,47 @@
+import {
+  isType
+} from 'graphql';
+import {
+  request,
+  express,
+  graphqlHTTP,
+  schema,
+  connection
+} from '../../../setup/test_helper'
+
+describe('user login', () => {
+  let server = express();
+
+  beforeAll(() => {
+    server.use(
+      '/',
+      graphqlHTTP({
+        schema: schema,
+        graphiql: false
+      })
+    )
+  })
+
+  afterAll(done => {
+    connection.close();
+    done();
+  });
+
+  it('can update user profile', async (done) => {
+    var email = 'test@test.com'
```

Comment:
I am not seeing address or image in here. Should that be a part of this? @GarrettCottrell  @jdcarey128  @the-color-bliu